packages/stacks-svelte/src/components/ExpandingInput/ExpandingInput.svelte

```diff
@@ -3,29 +3,36 @@
 </script>
 
 <script lang="ts">
-    /**
-     * The id to pair with a label
-     * @type {string} Id
-     */
-    export let id: string;
-
-    /**
-     * The input's variant
-     * @type {"default" | "chat"} Variant
-     */
-    export let variant: Variant = "default";
-
-    /**
-     * Initial value to pass to input
-     */
-    export let value = "";
+    interface Props {
+        /**
+         * The id to pair with a label
+         * @type {string} Id
+         */
+        id: string;
+
+        /**
+         * The input's variant
+         * @type {"default" | "chat"} Variant
+         */
+        variant?: Variant;
+
+        /**
+         * Initial value to pass to input
+         */
+        value?: string;
+
+        /**
+         * Additional input attributes like `placeholder`
+         */
+        [key: string]: unknown;
+    }
 
-    $: classes = getClasses(variant);
+    let { id, variant = "default", value = "", ...rest }: Props = $props();
 
     const getClasses = (variant: string) => {
         let common = "s-textarea ws-pre-wrap break-word hmx3 lh-lg";
         let textarea = `${common} ps-absolute i0 h100`;
-        let grower = `${common} v-hidden overflow-hidden`;
+        let grower = `${common} v-hidden overflow-hidden h100`;
```

Comment:
Seems like we can have `h100` as part of the `common` set of classes and not have to include it in `textarea` and `grower`